source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -31,18 +31,16 @@ There are 4 possible responses:
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-| Response Code  | Meaning                    |
-| -------------- | -------------------------- |
-|204             | Capture request accepted   |
-|404             | `paymentId` does not exist |
-|409             | Capture request rejected   |
-|500             | Downstream error           |
+|Response code | Meaning |
+|--|--|
+|204| Your capture request was accepted. |
+|404| No payment matched the `paymentId` you provided. |
+|409| The payment cannot be captured because its `status` is not `capturable`. |
+|500| Something is wrong with GOV.UK Pay. Please [contact us](/support_contact_and_more_information/#contact-us). |
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-Your service must send the delayed capture request within 120 hours (5 days) of the
-payment creation, regardless of how long your user takes to complete the
-payment flow. Otherwise, it will expire.
+Your service must send the delayed capture request within 120 hours (5 days) of creating the payment, regardless of how long your user takes to complete the payment flow. Otherwise [the payment will expire](https://docs.payments.service.gov.uk/payment_flow/#resuming-an-incomplete-payment).
```

Comment:
This link should be to https://docs.payments.service.gov.uk/payment_flow/#payment-expires.

I just checked that link and it mentions the standard 90 minute expiry, which is going to be confusing, so I think that will need some tweaks too.

Reply:
Sorry about that - fixed (and other changes discussed) in https://github.com/alphagov/pay-tech-docs/pull/363/commits/c94e1afb26f89ef2e9ae0117a19e3bd877329027.